solutions/community-deploy/communitybuild.sh

```diff
@@ -0,0 +1,133 @@
+#!/bin/bash
+
+usage() { 
+echo "Generate cortx-all docker image from provided CORTX release build"
```

Comment:
use `HEREDOC` for usage message instead of echo on each line.
Refer 
```
function usage() {
    cat << HEREDOC
Usage : $0 [--cortx-cluster, --destroy-cluster, --io-sanity, --support-bundle]
where,
    --cortx-cluster - Deploy CORTX Cluster on provided nodes.
    --destroy-cluster  - Destroy CORTX cluster.
    --io-sanity - Perform IO sanity test.
    --support-bundle - Collect support bundle logs.
HEREDOC
}
```

Reply:
I am not able to see you used `HEREDOC` here